Tests/conf.json

```diff
@@ -3188,6 +3200,456 @@
+        },
+        {
+            "playbookID": "Blueliv_ThreatContext_test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Blueliv ThreatContext"
+            ]
+        },
+        {
+            "playbookID": "Sophos Central Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Sophos Central"
+            ]
+        },
+        {
+            "playbookID": "Darktrace Test Playbook",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "Darktrace"
+            ]
+        },
+        {
+            "playbookID": "Recorded Future Test Playbook",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Recorded Future v2"
+            ]
+        },
+        {
+            "playbookID": "X509Certificate Test Playbook",
+            "fromversion": "6.0.0"
+        },
+        {
+            "playbookID": "Acalvio Sample Playbook",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Acalvio ShadowPlex"
+            ]
+        },
+        {
+            "playbookID": "XMCyberIntegration-Test",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "XMCyber"
+            ]
+        },
+        {
+            "playbookID": "3da36d51-3cdf-4120-882a-cee03b038b89",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "FortiManager"
+            ]
+        },
+        {
+            "playbookID": "Logzio - Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Logz.io"
+            ]
+        },
+        {
+            "playbookID": "Carbon Black Enterprise EDR Process Search Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Carbon Black Enterprise EDR"
+            ]
+        },
+        {
+            "playbookID": "playbook-SophosXGFirewall-test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "sophos_firewall"
+            ]
+        },
+        {
+            "playbookID": "playbook-Cyberint_Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "cyberint"
+            ]
+        },
+        {
+            "playbookID": "Druva-Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Druva Ransomware Response"
+            ]
+        },
+        {
+            "playbookID": "LogPoint SIEM Integration - Test Playbook 2",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "LogPoint SIEM Integration"
+            ]
+        },
+        {
+            "playbookID": "LogPoint SIEM Integration - Test Playbook 1",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "LogPoint SIEM Integration"
+            ]
+        },
+        {
+            "playbookID": "Polygon-Test",
```

Comment:
@ShahafBenYakir please go over this list and make sure that what @hod-alpert added makes sense